File: rest/outgoing-caller-ids/instance-post-example-1/instance-post-example-1.9.x.java
// Install the Java helper library from twilio.com/docs/java/install
import com.twilio.Twilio;
import com.twilio.rest.api.v2010.account.OutgoingCallerId;

public class Example {
  // Get your Account SID and Auth Token from https://twilio.com/console
  // To set up environment variables, see http://twil.io/secure
  public static final String ACCOUNT_SID = System.getenv("TWILIO_ACCOUNT_SID");
  public static final String AUTH_TOKEN = System.getenv("TWILIO_AUTH_TOKEN");

  public static void main(String[] args) {
    Twilio.init(ACCOUNT_SID, AUTH_TOKEN);

    // Get an object from its sid. If you do not have a sid,
    // check out the list resource examples on this page
    OutgoingCallerId callerId = OutgoingCallerId.updater("PNe536d32a3c49700934481addd5ce1659")
        .setFriendlyName("My Second Line")
        .update();

    System.out.println(callerId.getFriendlyName());
  }
}

File: rest/applications/instance-post-example-1/instance-post-example-1.9.x.py
# Download the Python helper library from twilio.com/docs/python/install
import os
from twilio.rest import Client

# Your Account Sid and Auth Token from twilio.com/user/account
# To set up environmental variables, see http://twil.io/secure
account_sid = os.environ['TWILIO_ACCOUNT_SID']
auth_token = os.environ['TWILIO_AUTH_TOKEN']
client = Client(account_sid, auth_token)

app = client.applications("AP2a0747eba6abf96b7e3c3ff0b4530f6e") \
            .update(voice_url="http://demo.twilio.com/docs/voice.xml",
                    sms_url="http://demo.twilio.com/docs/sms.xml")

print(app.voice_url)
